### PR TITLE
Rdm 9580

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
RDM-9580: Database Query Timeouts to be Revisited
      [RDM-9580] (https://tools.hmcts.net/jira/browse/RDM-9580).

### CODE REVIEW

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ java:
     DATA_STORE_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
     DATA_STORE_DB_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
-    DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
+    DATA_STORE_DB_OPTIONS: "?stringtype=unspecified&socketTimeout=25"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
```
Comment (Thor-tech-of-metal) [inline charts/ccd-data-store-api/values.preview.template.yaml:17]: 25 seconds , this value needs to be discussed with @MSancaktutar

File: charts/ccd-data-store-api/values.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ java:
     DATA_STORE_DB_NAME: ccd_data_store
     DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com
     DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-{{ .Values.global.environment }}
-    DATA_STORE_DB_OPTIONS: "?stringtype=unspecified&sslmode=require"
+    DATA_STORE_DB_OPTIONS: "?stringtype=unspecified&sslmode=require&socketTimeout=25"
     DATA_STORE_DB_MAX_POOL_SIZE: 48
```
Comment (Thor-tech-of-metal) [inline charts/ccd-data-store-api/values.yaml:27]: 25 seconds , this value needs to be discussed with @MSancaktutar

**Comment by Thor-tech-of-metal** [comment]
@MSancaktutar the application.propertiies has a default value for JDBC url connection parameters, Should We add a default value for socketTimeout there ?

**Comment by rajaram-nagarajan** [comment]
Stale

**Pull request closed without merging**